css/home.css

```diff
@@ -1,49 +1,171 @@
 /* Home page specific styles */
+@media only screen and (min-width: 360px) and (max-width: 960px) {
```

Comment:
Is it necessary for this line of code to be at the beginning?

Reply:
no is not, in the css style page we use first the mobile version of the page and later we add the media query in this case the media query is worng i will fixed.